examples/prompts/llm_monitoring/openai_proxy_quickstart.ipynb

```diff
@@ -225,18 +304,6 @@
     "r = openaimon.ChatCompletion.create(model=OPENAI_MODEL, messages=[{\"role\": \"user\", \"content\": f\"hello world!\"}])\n",
     "r = openaimon.ChatCompletion.create(model=OPENAI_MODEL, messages=[{\"role\": \"user\", \"content\": f\"what is 2+2?\"}])"
```

Comment:
minor: should we add the standard text here about how the user can create a board/link them to something?

Reply:
this init monitor should dump out a link to the table. I will leave it to you to add more instructions if needed (: